examples/interop/wampy-nodejs/callee.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"encoding/json"
+	"github.com/gammazero/nexus/client"
+	"github.com/gammazero/nexus/router"
+	"github.com/gammazero/nexus/wamp"
+	"io"
+	"log"
+	"os"
+	"os/signal"
+	"time"
+)
+
+const procedureName = "server.time"
+const wwsPort = 8008
+
+func main() {
+	broker := newBroker(wwsPort)
```

Comment:
`Broker` is probably not a good name here.  WAMP defines a `broker` role with functionality to handle pub/sub.  Calling you RPC server `broker` may confuse people who think your broker has something to do with the WAMP broker.

May I suggest calling this `timeServer`

Reply:
Agree on the confusion on the word `broker` (sorry for the novice mistake). I have updated the example, however, used the name `WampClient`